doc/reference/src/main/asciidoc/configuration.asciidoc

```diff
@@ -17,8 +17,7 @@ You should set the following required properties:
 [options="header"]
 |===============
 |Property = Example Value|Description
```

Comment:
Maybe this table (or the one below?) should also have a line for the `cdiLiteMode` flag?